astropy/io/ascii/latex.py

```diff
@@ -321,6 +321,16 @@ class AASTexHeaderSplitter(LatexSplitter):
     This splitter expects the following LaTeX code **in a single line**:
 
         \tablehead{\colhead{col1} & ... & \colhead{coln}}
+
+    If the \tablehead defines multiple output lines separated by \\
```

Comment:
I think that everything that's code (even LaTeX code) should be enclosed like this:
```
If the `\tablehead`
```
so that sphinx renders it nicely.